opentelemetry-api/src/opentelemetry/baggage/__init__.py

```diff
@@ -121,8 +121,8 @@ def _is_valid_value(value: object) -> bool:
     parts = str(value).split(";")
     is_valid_value = _VALUE_PATTERN.fullmatch(parts[0]) is not None
     if len(parts) > 1:  # one or more properties metadata
-        for property in parts[1:]:
-            if _PROPERT_PATTERN.fullmatch(property) is None:
+        for properties in parts[1:]:
```

Comment:
Maybe `prop` instead since this looks like is one property?

Reply:
Do I keep the change or revert it?

Reply:
I would use `prop`